src/assets/join.svg

```diff
@@ -0,0 +1 @@
.726-7.782-12.117-12.673-14.063-2.057-.834-4.002-1.278-5.892-1.39h-.778c-5.725 0-9.338 3.502-11.34 6.448-2.445 3.558-3.835 8.283-3.835 12.952 0 12.618 8.95 30.183 23.457 30.183 8.449 0 14.118-10.283 14.063-19.844 0-1.834-.167-3.669-.445-5.503" stroke="#181B2B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M43.417 23.234c-6.56 7.838-12.284 15.675-18.343 23.957-4.058 5.503-8.17 11.173-12.618 16.843-1.39 1.779-2.779 3.502-4.113 5.17a213.348 213.348 0 0 0-6.337 8.115.842.842 0 0 0-.055.945c.167.222.445.389.722.389h.167l59.865-10.672M18.626 75.318c-1.612-2.78-3.836-7.17-4.892-12.396" stroke="#181B2B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M34.134 73.094a.793.793 0 0 1-.667-.333c-5.391-7.726-8.504-16.842-8.504-25.07M55.756 45.19c.89-2.445 3.224-4.224 5.614-4.224 1.223 0 1.779.556 2.223.945.39.334.612.611 1.501.611.89 0 1.279-.555 1.946-1.667.778-1.223 1.667-2.78 4.002-2.78.166 0 .389.056.667.112.333.055.778.167 1.222.167 1.39 0 2.669-.779 3.447-2.002M63.926 58.865c-.278 0-.556-.112-.89-.278-.555-.278-1.556-.723-3.557-1.279-.389-.11-.667-.5-.611-.89a.842.842 0 0 1-.056-.944c.778-1.223 1.223-2.057 1.557-2.668.389-.778.667-1.279 1.223-1.39.61-.167 1.056.167 1.834.667.556.39 1.445.945 2.779 1.668.222.11.39.333.445.556a.91.91 0 0 1-.112.667c-1.056 1.556-1.39 2.445-1.5 2.89-.111.39-.278.778-.723.945-.111.056-.278.056-.389.056zM49.809 35.74c-1.557 0-2.78-1.166-2.78-2.723 0-1.5.945-3.557 2.668-3.39.945.11 1.78.555 2.446 1.39.334.444.556 1.11.556 1.778 0 1.445-1.167 2.946-2.89 2.946zM87.106 48.359c-.779 0-1.501-.611-2.724-1.612-.334-.278-.723-.611-1.167-.945a.792.792 0 0 1-.334-.667c-.055-.056-.111-.056-.167-.111-.333-.334-.278-.89.056-1.168 1.39-1.278 1.834-2.279 2.168-2.946.278-.5.5-1 1.056-1.167.722-.167 1.223.222 2.89 1.445.445.334 1 .779 1.668 1.223.333.223.444.723.222 1.056l-.5 1c-1.334 2.502-1.89 3.558-2.89 3.78-.056.112-.167.112-.278.112zM74.154 28.126c0-3.335 2.557-7.06 6.281-7.06 1.279 0 2.001.779 2.001 2.113 0 .277-.055.555-.055.833 0 .223-.056.445-.056.667 0 .89.778 1.279 1.056 1.279.834 0 1.501-1.279 2.168-2.557.945-1.834 2.112-4.113 4.614-4.113.833 0 1.5.722 1.5 1.556 0 .167 0 .39-.055.611 0 .187-.056.369-.056.556.89-.11 1.668-.945 2.39-1.723M65.871 16.397c-.167 0-.334-.055-.445-.11a17.46 17.46 0 0 0-1.056-.668c-1.278-.778-2.446-1.556-3.724-2.279-.39-.222-.556-.723-.334-1.112.056-.166.223-.278.334-.333.056-.167.167-.278.334-.39 1.612-1.945 2.334-3.112 2.723-3.89.39-.667.667-1.056 1.223-1.168.556-.11.89.223 1.39.612.611.556 1.723 1.5 4.224 2.78a.86.86 0 0 1 .445.61c.055.278 0 .5-.167.723-.056.056-4.169 4.836-4.28 4.947-.111.111-.222.223-.39.278h-.277zM83.548 8.06c-.89 0-1.723-.39-2.335-1.001-.667-.556-1.111-1.445-1.111-2.502C80.102 2.834 81.324 1 83.103 1h.556c1.667 0 3.28 1.779 3.28 3.613.055 2.112-1.724 3.446-3.391 3.446zM87.994 55.53l.5.166c1.557.556 5.17 1.835 6.837 1.835.167 0 .334 0 .556-.056" stroke="#181B2B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
Make this with dynamic colors for night mode
(Try to make screenshots for PR in 3 versions for such ui tasks: day & night mode for desktop, mobile) for decreasing future small fixes